src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -0,0 +1,296 @@
+    }
+
+    private static List<Allele> reverseComplementAlleles(final List<Allele> originalAlleles, final Interval target, final ReferenceSequence refSeq, final boolean isBiAllelicIndel, final boolean addToStart){
+        final List<Allele> alleles = new ArrayList<>();
+
+        for (final Allele oldAllele : originalAlleles) {
+            alleles.add(LiftoverUtils.reverseComplement(oldAllele, target, refSeq, isBiAllelicIndel, addToStart));
+        }
+
+        return alleles;
+    }
+
+    private static Allele reverseComplement(final Allele oldAllele, final Interval target, final ReferenceSequence referenceSequence, final boolean isBiAllelicIndel, final boolean addToStart){
+
+        if (oldAllele.isSymbolic() || oldAllele.isNoCall()) {
+            return oldAllele;
+        }
+        else if (isBiAllelicIndel) {
+            // target.getStart is 1-based, reference bases are 0-based
+            final StringBuilder alleleBuilder = new StringBuilder(target.getEnd() - target.getStart() + 1);
+
+            if (addToStart) {
+                alleleBuilder.append((char) referenceSequence.getBases()[target.getStart() - 2]);
+            }
+            alleleBuilder.append(SequenceUtil.reverseComplement(oldAllele.getBaseString().substring(1, oldAllele.length())));
+            if (!addToStart) {
+                alleleBuilder.append((char) referenceSequence.getBases()[target.getEnd() - 1]);
+            }
+
+            return Allele.create(alleleBuilder.toString(), oldAllele.isReference());
+        }
+        else {
+            return Allele.create(SequenceUtil.reverseComplement(oldAllele.getBaseString()), oldAllele.isReference());
+        }
+    }
+
+    protected static GenotypesContext fixGenotypes(final GenotypesContext originals, final List<Allele> originalAlleles, final List<Allele> newAlleles) {
+        // optimization: if nothing needs to be fixed then don't bother
+        if (originalAlleles.equals(newAlleles)) {
+            return originals;
+        }
+
+        if (originalAlleles.size() != newAlleles.size()) {
+            throw new IllegalStateException("Error in allele lists: the original and new allele lists are not the same length: " + originalAlleles.toString() + " / " + newAlleles.toString());
+        }
+
+        final Map<Allele, Allele> alleleMap = new HashMap<>();
+        for (int idx = 0; idx < originalAlleles.size(); idx++) {
+            alleleMap.put(originalAlleles.get(idx), newAlleles.get(idx));
+        }
+
+        final GenotypesContext fixedGenotypes = GenotypesContext.create(originals.size());
+        for (final Genotype genotype : originals) {
+            final List<Allele> fixedAlleles = new ArrayList<>();
+            for (final Allele allele : genotype.getAlleles()) {
+                if (allele.isSymbolic() || allele.isNoCall()) {
+                    fixedAlleles.add(allele);
+                }
+                else {
+                    Allele newAllele = alleleMap.get(allele);
```

Comment:
no need for this check

Reply:
symbolic probably not, but for NoCall I think we do, right?

Reply:
right, the NoCall is allowed to be in the genotype, while missing from the VC, but not the symbolic...so please remove the check for the symbolic allele as it is confusing.

Reply:
ok, removed it